README.md

```diff
@@ -2,10 +2,13 @@
 
 <!-- badges: start -->
 
-[![R-CMD-check-bioc-devel](https://github.com/microbiome/mia/workflows/R-CMD-check-bioc-devel/badge.svg)](https://github.com/microbiome/mia/actions)
-<!--[![R-CMD-check-bioc](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml/badge.svg)](https://github.com/microbiome/mia/actions/workflows/check-bioc.yml)-->
+[![Platforms](http://bioconductor.org/shields/availability/release/miaViz.svg)](https://bioconductor.org/packages/release/bioc/html/miaViz.html#archives)
+[![rworkflows](https://github.com/microbiome/mia/workflows/rworkflows/badge.svg)](https://github.com/microbiome/mia/actions)
```

Comment:
I am wondering if this badge shows the most recent action ran in mia repo. If so, then this takes into account also those ran in other branches in devel.

For instance, if action fails in "branch_X", that is not informative for user. --> Can you check if we can only show the action in devel branch?

Reply:
Fixed